scripts/orama-search/sync-orama-cloud.mjs

```diff
@@ -0,0 +1,77 @@
+import { siteContent } from './get-documents.mjs';
+
+// The following follows the instructions at https://docs.oramasearch.com/cloud/data-sources/custom-integrations/webhooks
+
+const INDEX_ID = process.env.ORAMA_INDEX_ID;
+const API_KEY = process.env.ORAMA_SECRET_KEY;
+const ORAMA_API_BASE_URL = `https://api.oramasearch.com/api/v1/webhooks/${INDEX_ID}`;
+
+const oramaHeaders = {
+  'Content-Type': 'application/json',
+  Authorization: `Bearer ${API_KEY}`,
+};
+
+// Orama allows to send several documents at once, so we batch them in groups of 50.
+// This is not strictly necessary, but it makes the process faster.
+const runUpdate = async () => {
+  const batchSize = 50;
+  const batches = [];
+
+  for (let i = 0; i < siteContent.length; i += batchSize) {
+    batches.push(siteContent.slice(i, i + batchSize));
+  }
+
+  console.log(
```

Comment:
Even if these logs are CI only, I'd say these are not needed. No human is going to read this, and if we ever need to read the source to understand, we just open the code.

Reply:
It just adds extra verbosity that is not needed.